sdk/react/panels/AudioAndCCSelectionPanel.js

```diff
@@ -56,13 +64,20 @@ var AudioAndCCSelectionPanel = React.createClass({
   },
 
   onAudioTrackSelected: function(name) {
-    if (this.props.selectedAudioTrackTitle !== name) {
-      this.props.onSelectAudioTrack(name);
+    let localizedTitleForUndefinedLanguage = Utils.localizedString(this.props.config.locale, stringConstants.undefinedLanguageTitle, this.props.config.localizableStrings);
+    let localizedTitleForNoLinguisticContent = Utils.localizedString(this.props.config.locale, stringConstants.noLinguisticContentTitle, this.props.config.localizableStrings);
+    var originalName = name;
+
+    originalName = originalName.replace(localizedTitleForUndefinedLanguage, stringConstants.undefinedLanguageTitle);
+    originalName = originalName.replace(localizedTitleForNoLinguisticContent, stringConstants.noLinguisticContentTitle);
+
+    if (this.props.selectedAudioTrackTitle !== originalName) {
+      this.props.onSelectAudioTrack(originalName);
```

Comment:
I cannot tell what the different is between onSelectAudioTrack and onAudioTrackSelected.  I can see what this function is doing but if I were to read these two function names someone else in the code, I would not know which one to use. Can you explain the difference and maybe we can name them better?

Reply:
We can do this in another PR.